### PR TITLE
Split the *_step part of cms/grading/__init__ to a new package

### PR DESCRIPTION
**Pull request by stefano-maggiolo**

### CODE REVIEW

**Comment by stefano-maggiolo** [comment]
Sorry, I should have said I didn't change anything apart from prepending some underscore, and some minor clarification in comments.

---

Review status: all files reviewed at latest revision, 9 unresolved discussions.

---

*[cms/grading/steps/compilation.py, line 48 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAlljO9W_3idAaWI6O:-LACUMiy6qO1hEMJFHwa:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/compilation.py#L48)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/string/strings/
</blockquote></details>

Done.

---

*[cms/grading/steps/evaluation.py, line 48 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAmqTX1SU_uNEvkRdL:-LACUPw05sxh9cMV7m_s:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/evaluation.py#L48)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/string/strings/
</blockquote></details>

Done.

---

*[cms/grading/steps/messages.py, line 53 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAm3NV7rKjWIUeioZ3:-LACUUOjBvXo68CTOx6J:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/messages.py#L53)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/unicode/str/ (x3)
</blockquote></details>

Done.

---

*[cms/grading/steps/trusted.py, line 50 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAn6We9XbHCpVoYWdd:-LACUXtE5CerEWARLc4W:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/trusted.py#L50)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/string/str/ (x2)
</blockquote></details>

Done.

---

*[cms/grading/steps/trusted.py, line 71 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAnCXe80JQsZ7oIQq-:-LACUbPxBSXXGaUkOnS9:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/trusted.py#L71)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/string/str/
</blockquote></details>

Done.

---

*[cms/grading/steps/whitediff.py, line 45 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAnc7i83-PeyLlfBQr:-LACUrnTC_s7w7cDfWs7:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/whitediff.py#L45)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

It took me a bit to understand what this meant, could you rephrase it as "... the Unicode White_Space characters (see ...) that are in the ASCII range."?
</blockquote></details>

Done.

---

*[cms/grading/steps/whitediff.py, line 63 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAnaDY4uWtZ2Sc7gOM:-LACUwOEDgLtcK233w9v:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/whitediff.py#L63)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/string/str/ (x2)
</blockquote></details>

Done.

---

*[cms/grading/steps/whitediff.py, line 134 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAnt0m-XJcc5yH3UpE:-LACUwqnBb2nPhQz7s99:b-896fix) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/whitediff.py#L134)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

s/string/str/ (x2) and s/unicode/str/ (x1)
</blockquote></details>

Done.

---

*[cms/grading/tasktypes/Communication.py, line 29 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAmUwA7XApRzLpaiWZ:-LACUxbfCsmZVQz5fBIx:bxffls1) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/tasktypes/Communication.py#L29)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

Was this for debugging? I think you should undo or explain why it's still needed.
</blockquote></details>

Ooops.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/cms-dev/cms/939#-:-LACUBjt7Ywlo-1-UT5v:b-8it47x)*

**Comment by lerks** [comment]
Reviewed 5 of 5 files at r3, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[cms/grading/steps/whitediff.py, line 134 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAnt0m-XJcc5yH3UpE:-LACWOsz56VXgDA0BcTI:bip26fi) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/whitediff.py#L134)):*
<details><summary><i>Previously, stefano-maggiolo (Stefano Maggiolo) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Missed the unicode

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/cms-dev/cms/939)*

**Comment by stefano-maggiolo** [comment]
Review status: 16 of 17 files reviewed at latest revision, 1 unresolved discussion.

---

*[cms/grading/steps/whitediff.py, line 134 at r1](https://beta.reviewable.io/reviews/cms-dev/cms/939#-LAAnt0m-XJcc5yH3UpE:-LACXM220lop7corTEow:b-3gzy7n) ([raw file](https://github.com/cms-dev/cms/blob/af59690cda374df31c005cda7785966416ce5e5c/cms/grading/steps/whitediff.py#L134)):*
<details><summary><i>Previously, lerks (Luca Wehrstedt) wrote…</i></summary><blockquote>

Missed the unicode
</blockquote></details>

Oh, this unicode

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/cms-dev/cms/939)*

**Comment by lerks** [comment]
Reviewed 1 of 1 files at r5.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/cms-dev/cms/939)*
